src/main/java/org/digma/intellij/plugin/assets/AssetsService.java

```diff
@@ -68,6 +69,7 @@ public String getAssets() {
 
     public void showAsset(String spanId) {
         Log.log(logger::debug, project, "showAsset called");
+        ActivityMonitor.getInstance(project).registerSpanLinkClicked(ActivityMonitor.Panel.Assets);
```

Comment:
if use our own name , it would be less verbose
```suggestion
        ActivityMonitor.getInstance(project).registerSpanLinkClicked(MonitoredPanel.Assets);
```